packages/vue/src/components/authenticator.vue

```diff
@@ -224,41 +208,19 @@ const onConfirmVerifyUserSubmitI = (e: Event) => {
   }
 };
 
-// watchers
-
-/**
- * Update service facade when context updates
- */
-
-const user: Ref<AmplifyUser | null> = ref(null);
-const signOut = ref();
```

Comment:
Now inferred directly to `useAuthenticator`, which resolves the race condition between getting `user` and evaluating `state.matches('authenticated').